### ISSUE TITLE
not working in Stable Diffusion WebUI Forge

### ISSUE THREAD

**Issue by nicegoodabc** [silveroxides/sd-webui-untitledmerger#5]
## Issue Description
The sd-webui-untitledmerger plugin works well in the standard Stable Diffusion WebUI, but it's not functioning correctly in Stable Diffusion WebUI Forge.

## Specific Problem
When using the plugin in Stable Diffusion WebUI Forge, the merged models are not taking effect, regardless of how they are combined.

## Expected Behavior
The plugin should work in Stable Diffusion WebUI Forge as it does in the standard Stable Diffusion WebUI, allowing for effective model merging.

## Current Behavior
The merged models have no effect on the output in Stable Diffusion WebUI Forge.

**Comment by silveroxides** [comment]
Yes and it won't be either because it would require so many breaking changes.

It works with A1111 and ReForge.
Forge repo has switched directions and only reason to use it instead of ReForge is if one want to use Flux1 but even then you might as well switch to ComfyUI because Flux1 performance in Forge is trash.

**Comment by duskfallcrew** [comment]
It wasn't working on forge on recent commits but as Silver stated i was looking into forkign it because i literally cannot bring myself to use comfyUI that much -- not that ComfyUI SUCKS it's amazing and pwoerful and for FORGE sure i'm sure forge/otherwise isn't perfect but i don't use forge -- 

I uh.
Present you this: 
https://github.com/Ktiseos-Nyx/batteriesincluded-merger
Warning: I forgot to name it PROPERLY. this is the NON FORK REPO, because i don't like having to commit to branches adn going "PR TO MAIN" and find out i'm Pr'ing to someone else's repo when i've tried to change 90% of it lol. 

Silver's still heavily in the code base -- welcome to still mess with it, but there must've been some code API changes that made it not work in forge after a while.  Back like in Feb, i can see why it might've worked but forge made a bujnch of changes after i started using forge and it just wasn't working again... I am not done with this clearly. xD

**Comment by silveroxides** [comment]
@duskfallcrew 
As long as it works then great. 

There is a port of it that is WIP in https://github.com/silveroxides/ComfyUI-ModelUtils for ComfyUI now too for those who want simplistic merging of all kinds of model types
